### DISCUSSION TITLE
I'm using FusionCache!

### DISCUSSION THREAD

**Discussion by tesfayeKebebe** [ZiggyCreatures/FusionCache#295] [Show and tell]
## Scenario
Describe how you are using FusionCache: commercial product, oss project, monolith, microservice, web, mobile app, CLI, etc

## Liked
What you liked, features, docs or anything

## Unliked/Missing
Things you didn't like or felt was missing

## Closing Thoughts
Any closing thoughts

**Comment by jodydonetti** [comment]
Thanks, I'm glad you're using FusionCache! But can you fill it out 😅 ?
